test/e2e/operator_test.go

```diff
@@ -2642,10 +2642,17 @@ func TestAWSELBConnectionIdleTimeout(t *testing.T) {
 		t.Fatalf("expected %s=%s, found %s=%s", key, expected, key, v)
 	}
 
-	// Make sure we can resolve the route's host name.  It may take some
-	// time for the ingresscontroller's wildcard DNS record to propagate.
+	// Get the ELB's hostname via the wildcard DNS record
+	wildcardRecordName := controller.WildcardDNSRecordName(ic)
+	wildcardRecord := &iov1.DNSRecord{}
+	if err := kclient.Get(context.TODO(), wildcardRecordName, wildcardRecord); err != nil {
+		t.Fatalf("failed to get wildcard dnsrecord %s: %v", wildcardRecordName, err)
+	}
+	elbHostname := wildcardRecord.Spec.Targets[0]
+
+	// Wait until we can resolve the ELB's hostname
 	if err := wait.PollImmediate(5*time.Second, 5*time.Minute, func() (bool, error) {
-		_, err := net.LookupIP(route.Spec.Host)
+		_, err := net.LookupIP(elbHostname)
```

Comment:
We actually set the route name and create it in https://github.com/openshift/cluster-ingress-operator/pull/944/files#diff-cf4b4e5424070a666a364d1d1b04011478d888d6d3d65c943ca7783333b7a6e4R2616.  

I know this issue is seen on platforms going back to 4.11, but did you investigate whether or not we need to wait for the route to be ready after we create it?